mod-source-record-manager-server/src/test/java/org/folio/rest/impl/changeManager/ChangeManagerAPITest.java

```diff
@@ -1032,39 +1012,28 @@ public void shouldProcessChunkOfRawRecordsWithDuplicatedTags(TestContext testCon
     async = testContext.async();
     RestAssured.given()
       .spec(spec)
-      .body(rawRecordsDto_2)
+      .body(rawRecordsDto_2.withId(UUID.randomUUID().toString()))
       .when()
       .post(JOB_EXECUTION_PATH + jobExec.getId() + RECORDS_PATH)
       .then()
       .statusCode(HttpStatus.SC_NO_CONTENT);
     async.complete();
 
-    String requestBody = findAll(putRequestedFor(urlEqualTo(PARSED_RECORDS_COLLECTION_URL))).get(0).getBodyAsString();
-    RecordCollection recordCollection = new ObjectMapper().readValue(requestBody, RecordCollection.class);
-    assertThat(recordCollection.getRecords().size(), is(50));
-
-    recordCollection.getRecords().forEach(record -> {
-      Assert.assertNotNull(record.getExternalIdsHolder());
-      Assert.assertNotNull(record.getExternalIdsHolder().getInstanceId());
-    });
-
-
     async = testContext.async();
     RestAssured.given()
       .spec(spec)
       .when()
       .get(JOB_EXECUTION_PATH + jobExec.getId())
       .then()
       .statusCode(HttpStatus.SC_OK)
-      .body("status", is(JobExecution.Status.COMMITTED.name()))
+      .body("status", is(JobExecution.Status.PARSING_IN_PROGRESS.name()))
```

Comment:
It takes time to get "COMMITTED" status

Reply:
Can be situation that for some test execution tries it would be enough time and status would be COMMITTED? So it can be a reason of passing this test time to time. In this case maybe need to delete this checking

Reply:
I decided remove these lines.